### ISSUE TITLE
Preloader 💫

### ISSUE THREAD

**Issue by Praful932** [Praful932/Kitabe#43]
**Is your feature request related to a problem? Please describe.**
Since the project is hosted on the heroku free tier, it takes some time to load, It would be nice to have a preloader.

**Describe the solution you'd like**
Before the homepage is loaded, there should be a loading animation whose design matches with the theme of the website. It should load only once(using cookies) when it enters through the domain name, and not everytime the user goes to the homepage.

- First a mockup design should be shown
- Then after approval code changes should be made

**Describe alternatives you've considered**
Open to alternatives!

**Additional context**
The preloader should be responsive and bug-free.

**Comment by cryptofox17** [comment]
Could I work on this issue?

**Comment by padmajabhol** [comment]
I want to work.

**Comment by maneprajakta** [comment]
@cryptofox17 go ahead with this.

**Comment by cryptofox17** [comment]
@Praful932 @maneprajakta i have made the animation, please look at it and let know if it needs any change. I have created a svg for the book and the animation was done using css. I have ensured to make it responsive and tried to match with the website theme. If you approve it, I'll fork and add this.
![Screen Shot 2021-03-12 at 19 18 41](https://user-images.githubusercontent.com/64403591/110949224-5cb2ef00-8368-11eb-8410-520144faba47.png)

https://user-images.githubusercontent.com/64403591/110949492-ad2a4c80-8368-11eb-9d9e-693fba9cddb0.mp4

**Comment by maneprajakta** [comment]
@cryptofox17 Seems cool to me.
Approved from my side :rocket: . 
Once go through our [Starter Guide](https://github.com/Praful932/Kitabe/discussions/42) it will help you in contributing to the project.

**Comment by Praful932** [comment]
@cryptofox17 Also make sure to only show the preloader once per window, Use Cookies 😉

**Comment by cryptofox17** [comment]
@Praful932  I am ready to make a PR for this issue but before that I wanted to ask if what I did is what you expected. 

I added the code for preloader in the index.html ensuring to use cookies. So if a user visits the website they'd be able to see it, but if they go to some other page or click on the logo to return to home page they wouldn't be able to see it. 
Is that enough?

**Comment by Praful932** [comment]
@cryptofox17 Yes, that's perfect from your description. If there any changes we will let you know in the PR :)
